testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/ConfigurationTools.cs

```diff
@@ -0,0 +1,92 @@
+using UnityEngine;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Text;
+
+namespace Unity.Netcode.MultiprocessRuntimeTests
+{
+    public class ConfigurationTools
+    {
+        public static JobQueueItemArray GetRemoteConfig()
```

Comment:
any reason the post uses async await but not the get?

Reply:
Not really, I am equalizing them now.

Reply:
oh, I would have expected the other way actually and use the language features. Those should work fine, no? UGS uses async await everywhere for their own SDK.

Reply:
It'd make following the sequence easier and would help with bubbling up exceptions

Reply:
Ok, I made another edit.